<?xml version="1.0"?>
<launch>
  <node name="visualize_models" pkg="ros_votegrasp" type="model2scene_node" cwd="node">
  <param name="model_dir" value="$(find ros_votegrasp)/data/ycb-models/"/>
  <param name="object_pose_path" value="/home/hoang/OSS/VoteGrasp/blender-scripts/scene_object_pose.txt"/> 
  <param name="grasp_save_path" value="/home/hoang/OSS/VoteGrasp/blender-scripts/scene_grasp.txt"/>
  <param name="model_save_path" value="/home/hoang/OSS/VoteGrasp/blender-scripts/pointcloud/models.ply"/>

  </node>
  <!-- Show in Rviz   -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find ros_votegrasp)/launch/votegrasp.rviz"/>
  
</launch>